internal/querycoord/querynode.go

```diff
@@ -335,7 +336,10 @@ func (qn *queryNode) getNodeInfo() (Node, error) {
 	if err != nil {
 		return nil, err
 	}
-	resp, err := qn.client.GetMetrics(qn.ctx, req)
+
+	ctx, cancel := context.WithTimeout(qn.ctx, time.Second)
```

Comment:
NIT: magic number for timeout duration. Maybe put it into configuration or const in the future.